### ISSUE TITLE
Export .about.yml files to project repositories?

### ISSUE THREAD

**Issue by toolness** [18F/projects#18]
@ccostino mentioned that the [`about.yml` specification](https://github.com/18F/about_yml) and the convention of adding `.about.yml` files to project repositories is actually part of the [ATO checklist](https://pages.18f.gov/before-you-ship/ato/checklist/) for projects.

This means that if we store the canonical data in the `projects` db instead of `.about.yml` files, we might mess up the ATO process.

One option is to change the ATO process, which could be hard.

Another option, though, is to actually have `projects` auto-generate `.about.yml` files and commit them to project repositories. These auto-generated files can have comments on the top instructing readers to not modify them directly, but instead edit the data via the `projects` admin UI.

Thoughts on this @gboone @brendansudol?

**Comment by brendansudol** [comment]
Love that idea so much, @toolness! checks all the right boxes for me :)

**Comment by ccostino** [comment]
@toolness, that would be awesome if there were a tool to help build and validate the `.about.yml` files in an interface for folks rather than hand-crafting and modifying YAML files!  Several of us have talked about this before in the past, and the validation and schema components already exist with [about_yml gem](https://github.com/18F/about_yml) and [about-yml-validator package](https://github.com/18F/about-yml-validator).  These could serve as a head start.

@jeremiak, @ertzeid, and @mtorres253, if any of you have a moment I think you might be able to weigh in a bit more on this for additional background and thoughts. :-)

**Comment by gboone** [comment]
I've long been a fan of combining the validator with some kind of webform to help users write their about.yml files correctly the first time _and_ easily commit them to the repo. Others will have more thoughts on how this has been approached in the past.

**Comment by ertzeid** [comment]
Yeah, I've had similar thoughts to Greg's -- anything that helps the
project owners do it right in the first place.

The Rails validators should make it a lot easier to also rectify any errors
in the .about.yml files, though IIRC I added those to the people files
(amanda.robinson.yml, etc., in the team-api.18f.gov repo), and the
validations may still need to be added for .about.yml files.

**Comment by brendansudol** [comment]
Nice, thanks for context all; I think the web form within the django admin within this app should do the trick here and then we can build a nice big button to sync them with their respective repos
